<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="EnqueueFrontButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>добавить в начало </string>
    </property>
   </widget>
   <widget class="QPushButton" name="DequeueFrontButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>удалить в начале очереди</string>
    </property>
   </widget>
   <widget class="QListWidget" name="queueListWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>256</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputTextEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Введите элемент в очередь</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>31</y>
      <width>731</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ConvertToCircularButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Превращаем в кольцо</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EnqueueBackButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>добавить в конец</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DequeueBackButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>410</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>удалить в конце очереди</string>
    </property>
   </widget>
   <zorder>listWidget</zorder>
   <zorder>EnqueueFrontButton</zorder>
   <zorder>DequeueFrontButton</zorder>
   <zorder>queueListWidget</zorder>
   <zorder>inputTextEdit</zorder>
   <zorder>ConvertToCircularButton</zorder>
   <zorder>EnqueueBackButton</zorder>
   <zorder>DequeueBackButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
